.github/workflows/test.yml

```
@@ -135,6 +135,23 @@ jobs:
           fi
         env:
           RUFF_VERSION: ${{ steps.ruff-action.outputs.ruff-version }}
+  test-default-version-from-requirements:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use default version from requirements.txt
+        id: ruff-action
+        uses: ./
+        with:
+          src: __tests__/fixtures/
```

```suggestion
          src: __tests__/fixtures/python-project
```
Set `src` to a path where there is nothing to lint/fix

---

Done